tests/sqlparser_common.rs

```diff
@@ -4758,6 +4758,25 @@ fn parse_alter_table() {
     }
 }
 
+#[test]
+fn alter_table_span_includes_semicolon() {
+    use sqlparser::ast::Spanned;
+    use sqlparser::dialect::PostgreSqlDialect;
+    use sqlparser::parser::Parser;
+
+    let sql = r#"-- foo
+ALTER TABLE users
+  ADD COLUMN foo
+  varchar; -- hi there"#;
+    let dialect = PostgreSqlDialect {};
+    let ast = Parser::parse_sql(&dialect, sql).unwrap();
+    let stmt = &ast[0];
+    let span = stmt.span();
+
+    assert_eq!(span.end.line, 4);
+    assert_eq!(span.end.column, 11);
+}
+
```

Comment:
this seems like a similar test to the one in spans.rs, the latter felt like it should provide sufficient coverage so that we can likely skip this one?